packages/ckeditor5-image/tests/manual/imageinsert.js

```diff
@@ -13,41 +13,50 @@ import CKFinder from '@ckeditor/ckeditor5-ckfinder/src/ckfinder';
 import ImageInsert from '../../src/imageinsert';
 import AutoImage from '../../src/autoimage';
 
-ClassicEditor
-	.create( document.querySelector( '#editor' ), {
-		plugins: [ ArticlePluginSet, ImageInsert, AutoImage, LinkImage, CKFinderUploadAdapter, CKFinder ],
-		toolbar: [
-			'heading',
-			'|',
-			'bold',
-			'italic',
-			'link',
-			'bulletedList',
-			'numberedList',
-			'blockQuote',
-			'insertImage',
-			'insertTable',
-			'mediaEmbed',
-			'undo',
-			'redo'
-		],
-		image: {
-			toolbar: [ 'imageStyle:inline', 'imageStyle:block', 'imageStyle:side', '|', 'toggleImageCaption', 'imageTextAlternative' ],
-			insert: {
-				integrations: [
-					'insertImageViaUrl',
-					'openCKFinder'
-				]
+createEditor( 'editor1', 'auto' );
+createEditor( 'editor2', 'block' );
+createEditor( 'editor3', 'inline' );
+
+function createEditor( elementId, imageType ) {
+	ClassicEditor
+		.create( document.querySelector( '#' + elementId ), {
+			plugins: [ ArticlePluginSet, ImageInsert, AutoImage, LinkImage, CKFinderUploadAdapter, CKFinder ],
+			toolbar: [
+				'heading',
+				'|',
+				'bold',
+				'italic',
+				'link',
+				'bulletedList',
+				'numberedList',
+				'blockQuote',
+				'insertImage',
+				'insertTable',
+				'mediaEmbed',
+				'undo',
+				'redo'
+			],
+			image: {
+				toolbar: [ 'imageStyle:inline', 'imageStyle:block', 'imageStyle:side', '|', 'toggleImageCaption', 'imageTextAlternative' ],
+				insert: {
+					integrations: [
+						'insertImageViaUrl',
+						'openCKFinder'
+					],
+					type: imageType
+				}
+			},
+			ckfinder: {
+				// eslint-disable-next-line max-len
+				uploadUrl: 'https://ckeditor.com/apps/ckfinder/3.5.0/core/connector/php/connector.php?command=QuickUpload&type=Files&responseType=json'
 			}
-		},
-		ckfinder: {
-			// eslint-disable-next-line max-len
-			uploadUrl: 'https://ckeditor.com/apps/ckfinder/3.5.0/core/connector/php/connector.php?command=QuickUpload&type=Files&responseType=json'
-		}
-	} )
-	.then( editor => {
-		window.editor = editor;
-	} )
-	.catch( err => {
-		console.error( err.stack );
-	} );
+		} )
+		.then( editor => {
+			window[ elementId ] = editor;
+
+			CKEditorInspector.attach( imageType, editor );
```

Comment:
Did you also notice the warnings caused by this line like below?
`inspector.js:2 [CKEditorInspector] The CKEditorInspector.attach( 'block', editor ) syntax has been deprecated and will be removed in the near future. To pass a name of an editor instance, use CKEditorInspector.attach( { 'block': editor } ) instead. Learn more in https://github.com/ckeditor/ckeditor5-inspector/blob/master/README.md.`